dev/d11de74d-0a9f-41e7-8b37-93d2a14ba697/device.lp

```diff
@@ -0,0 +1,20 @@
+(librepcb_device d11de74d-0a9f-41e7-8b37-93d2a14ba697
+ (name "LM393N")
+ (description "LM393N low-power, low-offset voltage dual comparator in a PDIP-8 package.")
+ (keywords "LM193,LM193N,LM193-N")
```

Comment:
Thanks for the update! Ugh, it seems that when I proposed the change, I accidentally used the term `193` instead of `393` :see_no_evil: Can you click one more time to accept the fix? Sorry for that :slightly_smiling_face: 

```suggestion
 (keywords "LM393,LM393N,LM393-N")
```

Reply:
Missed that as well...